Content.Server/GameObjects/Components/PlaceableSurfaceComponent.cs

```diff
@@ -12,38 +13,74 @@ namespace Content.Server.GameObjects.Components
     [ComponentReference(typeof(SharedPlaceableSurfaceComponent))]
     public class PlaceableSurfaceComponent : SharedPlaceableSurfaceComponent, IInteractUsing
     {
-        private bool _isPlaceable;
+        private bool isPlaceable;
+        private bool placeCentered;
+        private Vector2 positionOffset;
```

Comment:
Private fields should have a `_` prefix.

Reply:
Done, thanks a lot for the review.